apps/dragboard/lib.js

```diff
@@ -243,7 +224,34 @@ exports.input = function(options) {
             }
           }
         }
-      }
+      };
+
+  let catchSwipe = (_,__)=>{
```

Comment:
@gfwilliams, mostly out of curiosity, would it be smarter to just do `let catchSwipe = ()=>(` here, without the arguments? If I get to add this to other keyboards I can do whatever is better also.

Reply:
Yes, if you're not using the arguments it's better not to add them - it's not much but each argument you don't define saves you one variable's worth of memory